src/main/java/org/folio/rmapi/model/RootProxyCustomLabels.java

```diff
@@ -14,8 +14,11 @@
 
   @JsonProperty("proxy")
   private Proxy proxy;
-  
+
+  @JsonProperty("vendorId")
+  private String vendorId;
+
   @JsonProperty("labels")
   private List<CustomLabel> labelList;
-  
+
```

Comment:
Was wondering on the name of this model -- it is now proxy, custom labels and vendor id (and will also have ftf tokens). It seems like it is a customer configuration.

Reply:
Yes -- I agree - https://developer.ebsco.com/reference/rmapi#customlabelsproxy